### ISSUE TITLE
frontend losing state

### ISSUE THREAD

**Issue by colinallen** [inpho/inphosemantics-frontend#23]
Make some selections and load the data.
Hit 'Make selections' again and change value in number of results and hit enter while browser focus is still in the number box.

Results:

In Safari: all the other values are cleared (= reset)

In Firefox, things sometimes things go wrong, but I can't figure out exactly how. Sometimes the data is reloaded, but the next time you hit 'Make selections', all the values have been cleared; other times it loads a blank page. (You may have to cycle through 'make selections' a few times to see this.)

Values are also not being preserved when 'Export data' is selected, but I haven't opened a separate ticket on this because perhaps that part wasn't intended to be functional yet, and in any case it may be the same underlying problem.

**Comment by srwaggon** [comment]
This appears to be two separate issues.

The first issue -- that the searchLimit input box receives the 'enter' keydown and resets the form has been fixed: The input box now ignores submission on "enter" keydown.

I can't fully identify the second problem, but I have noticed that if there is insufficient data for a fully-formed query, then upon submitting the requests, the page turns blank and the modal clears its selections (but retains the available models/corpora).
